source/tyr/tests/integration/datapost_test.py

```diff
@@ -0,0 +1,37 @@
+from tests.check_utils import api_get, api_post, api_delete, api_put, _dt
+import json
+import pytest
+import os
+from navitiacommon import models
+from tyr import app
+import logging
+
+@pytest.fixture(scope="module")
+def create_instance_fr():
+    with app.app_context():
+        instance = models.Instance('fr')
+        models.db.session.add(instance)
+        models.db.session.commit()
+        return instance.id
+
+def test_post_pbf(create_instance_fr):
+    filename = 'empty_pbf.osm.pbf'
+    path = os.path.join(os.path.dirname(os.path.dirname(os.path.dirname(__file__))), 'tests/fixtures/', filename)
+    files = {'file': (open(path, 'rb'), filename)}
+
+    tester = app.test_client()
+    resp = tester.post('/v0/data/instances/fr', data=files)
+    assert resp.status_code == 200
+    assert os.path.isfile('/tmp/empty_pbf.osm.pbf')
+    os.remove('/tmp/empty_pbf.osm.pbf')
+    assert (not os.path.isfile('/tmp/empty_pbf.osm.pbf'))
```

Comment:
You should assert that the file doesn't exist at first, we want to be sure that we are the one having created this file